it-tests/BasicFlow.test.ts

```diff
@@ -151,10 +151,11 @@ async function createNewRoute(driver: WebDriver) {
 
 async function addActiveMQStep(driver: WebDriver) {
   await driver.wait(
-    until.elementLocated(By.xpath('//*[@data-type="node"]//*[@class="pf-topology__node__action-icon"]'))
+    until.elementLocated(By.css('g[data-testid^="custom-node__timer"]'))
   );
-  const threeDotsIconOfOneOfTheSteps = (await driver.findElements(By.xpath('//*[@data-type="node"]//*[@class="pf-topology__node__action-icon"]')))[1];
-  await threeDotsIconOfOneOfTheSteps.click();
+
+  const canvasNode = await driver.findElement(By.css('g[data-testid^="custom-node__timer"]'));
+  driver.actions().contextClick(canvasNode).perform();
```

Comment:
Since the 3-dots button is no longer there, we're triggering a right-click event over the node to get the same menu.

Reply:
just a small one for the stabillity.. is the "driver.actions...perform()" a floating Promise? I think we could missing 'await' here

Reply:
I don't know, I'll check

Reply:
Alrighty, I was missing the `await`, `perform` is certainly a `Promise`, thanks for the catch @djelinek